package.json

```diff
@@ -9,10 +9,12 @@
     "apollo-server": "^2.24.1",
     "csv-parser": "^3.0.0",
     "graphql": "^15.5.0",
+    "lint": "^0.7.0",
```

Comment:
uh, nope, that's not a dependency we want. I think you meant to add a lint SCRIPT that does `eslint .`